### PR TITLE
Sdit 1510 charges test

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingService.kt [hunk 1]
```diff
@@ -53,18 +55,24 @@ class CourtSentencingService(
...
             CourtAppearanceMappingDto(
               dpsCourtAppearanceId = courtCase.latestAppearance.appearanceUuid.toString(),
-              nomisCourtAppearanceId = nomisResponse.courtAppearanceIds.first().id,
-              nomisNextCourtAppearanceId = nomisResponse.courtAppearanceIds.first().nextCourtAppearanceId,
+              nomisCourtAppearanceId = nomisCourtAppearanceResponse.id,
+              nomisNextCourtAppearanceId = nomisCourtAppearanceResponse.nextCourtAppearanceId,
             ),
           ),
-          courtCharges = listOf(),
+          courtCharges = courtCase.latestAppearance.charges.mapIndexed { index, dpsCharge ->
```
Comment (lbennett-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtSentencingService.kt:70]: this is all dependent on nomis returning the charge Ids in the order they were sent to be created

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt [hunk 2]
```diff
@@ -117,6 +127,38 @@ class CourtCasesToNomisIntTest : SqsIntegrationTestBase() {
                     NOMIS_NEXT_COURT_APPEARANCE_ID.toString(),
                   ),
                 ),
+              )
+              .withRequestBody(
+                WireMock.matchingJsonPath(
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt:132]: very, very minor - but worth doing a static import of `Wiremock.equalTo` to reduce the noice in this code

**Review by andymarke — APPROVED** [review]
lgtm

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt (hunk 2 above)
Comment (lbennett-moj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/courtsentencing/CourtCasesToNomisIntTest.kt:132]: yes, must have been missed

**Pull request merged**